jsonschema-validation.xml

```diff
@@ -1316,11 +1316,11 @@
             &RFC6901;
             &RFC8259;
             <reference anchor="ecma262"
-            target="http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf">
+            target="https://www.ecma-international.org/ecma-262/11.0">
```

Comment:
Their official page at https://www.ecma-international.org/publications/standards/Ecma-262.htm points to the URL, "https://www.ecma-international.org/ecma-262/11.0/index.html#title".